### ISSUE TITLE
Discord notice in `Contributing.rst` is misleading

### ISSUE THREAD

**Issue by ghost** [SConsProject/scons#4632]
![screen-1731402599](https://github.com/user-attachments/assets/43ff6005-52ba-4e67-88b0-2750635e9a58)

https://github.com/SCons/scons/blob/b4d72c716aa98d3cefcf9fcb358429b08ac58114/CONTRIBUTING.rst?plain=1#L16-L22

**Comment by bdbaddog** [comment]
Feel free to PR if you feel this language isn't clear enough.

**Comment by ghost** [comment]
Regular discord user is registered discord user in my opinion... And I am bad at descriptive and sound strings of text so... And tbh I am even not sure what was meant in here. Something like we have discord is ok but everything else?

**Comment by mwichmann** [comment]
Will be happy to think about different wording if had a better idea what was confusing about this. 

The "project description" files like the READMEs and so forth haven't had as much editing attention as the SCons documentation.

**Comment by ghost** [comment]
> Will be happy to think about different wording if had a better idea what was confusing about this.

My interpretation of what I read: "We have discord. You doesn't need to be register in discord to contact us. Oh and discord also have a web interface"

But when I open it I see that I must register to write something so...

**Comment by mwichmann** [comment]
my system has gone wonky so I can't really check if the web interface still works (it keeps redirecting me). may be that commenting is no longer possible without being "signed in", in which case we should take that out.

**Comment by bdbaddog** [comment]
@dustdfg - please suggest other language if you'd like to see a change here. You're the first user who was confused by the language (or at least the first that let us know). Pretty sure "regular" meant to mean an installed app and not a web page..

**Comment by mwichmann** [comment]
I guess we should check if the web interface (a) still works at all and (b) allows writing to an SCons server channel without being logged in, and if not, removing the mention of the web interface.  My belief, which I can't easily test because my browser "helpfully" vectors over to the app due to some remembered information, is that you can use it, but login is required to post...

**Comment by ghost** [comment]
> I guess we should check if the web interface (a) still works at all and

It works, at least for me. The initial screenshot is a screenshot of what I saw in web version

**Comment by kprussing** [comment]
For reference, I use the web interface to participate, but I did have to make an account.

**Comment by dnwillia** [comment]
Yup, same here, web interface works totally fine.  You definitely need an account so that last sentence could just be removed entirely.

**Comment by mwichmann** [comment]
That file has been adjusted a couple of times, so I'm going to go ahead and close. Feel free to reopen if it's still misleading.

**Comment by dnwillia** [comment]
Hi Mats, I think if you just delete the last sentence then this is resolved.  i.e. remove this "You don't have to be a regular Discord user, there is a web interface."

**Comment by bdbaddog** [comment]
@dnwillia  it's already been changed to the following. Which is sufficient to close this issue.

```
You don't have to use the Discord app,
as there is a web interface (does require an account).
```

**Comment by dnwillia** [comment]
Ah, yes, did not see that.
